roachpb/data_test.go

```diff
@@ -534,13 +564,13 @@ func TestMakePriorityLimits(t *testing.T) {
 		math.MaxFloat64,
 	}
 	const trials = 100
-	for i, userPri := range userPs {
+	for _, userPri := range userPs {
 		seen := map[int32]struct{}{} // set of priorities
 		for j := 0; j < trials; j++ {
 			seen[MakePriority(userPri)] = struct{}{}
 		}
-		if len(seen) < 90 {
-			t.Errorf("%d: expected randomized values, got %v", i, seen)
+		if len(seen) < 5 {
```

Comment:
Resolution in distinct random values seems too attenuated here to be useful. See my previous comment on changing min and max user priority constants.